docs/usage/administration/Synapse-database-maintenance-tools.md

```diff
@@ -0,0 +1,36 @@
+This blog post by Victor Berger explains how to use many of the tools listed on this page: https://levans.fr/shrink-synapse-database.html
+
+# List of useful tools and scripts for maintenance Synapse database:
+
+## [Purge Remote Media API](https://github.com/matrix-org/synapse/blob/master/docs/admin_api/purge_remote_media.rst)
+The purge remote media API allows server admins to purge old cached remote media.
+
+#### Purge local media 
+
+There is no purge API for local media because you may be the only one with a copy. If you are sure you want to delete local media you could use something like the following to delete media that hasn't been accessed in a while.
```

Comment:
This is not up to date.
There is an API: https://matrix-org.github.io/synapse/latest/admin_api/media_admin_api.html#delete-local-media